test/timers/test-timers-reliability.js

```diff
@@ -7,7 +7,13 @@ var Timer = process.binding('timer_wrap').Timer;
 var assert = require('assert');
 
 var timerFired = false;
+var zeroTimerFired = false;
```

Comment:
Instead of having a bunch of named variables, couldn't we just use `common.mustCall()` to verify the function calls?